src/components/TodoComponents/Todo.css

```diff
@@ -0,0 +1,3 @@
+.task .done {
```

Comment:
Here our class refers to an element with a class of "done" who is a child of an element of class "task". Instead, we are referring to the same element so we would use `.task.done`. 👍